benchmarks/versus-external-pools/bench.sh

```diff
@@ -0,0 +1,37 @@
+export NODE_ENV=production
+
+# Execute bench
+# export POOL_SIZE=8
+# export NUM_ITERATIONS=10000
+# hyperfine --export-markdown BENCH-10000.MD --min-runs 10 \
+#   'node dynamic-poolifier.js' \
+#   'node dynamic-suchmokuo-node-worker-threads-pool.js' \
+#   'node fixed-poolifier.js' \
+#   'node static-suchmokuo-node-worker-threads-pool.js' \
+#   'node piscina.js'
+
+# echo "Sleeping...."
+# sleep 60
+
+export POOL_SIZE=10
+export NUM_ITERATIONS=100000
+hyperfine --export-markdown BENCH-100000.MD --min-runs 10 \
```

Comment:
I cannot perform this script cause I don't have `hyperfine` installed

Reply:
Unless you are on Windows, that's pretty easy to install: https://github.com/sharkdp/hyperfine

Reply:
Errata: it's also easily installable on windows with chocolatey.

Reply:
Added documentation on how to run bench